tests/test_transform.py

```diff
@@ -39,3 +45,24 @@ def test_pairwiseSeq_2():
     test_list = [1, 2, 3, 4]
     result = distill.pairwiseSeq(test_list, split=True)
     assert result == ((1, 2, 3), (2, 3, 4))
+    
+data = testing_utils.setup(os.path.join(DATA_DIR, "sample_data.json"), "integer")
+logs = data[1]
+
+def input_rule(log):
+    return "target" in log and "input" in log["target"]
+
+def test_label_features_1():
+    result = distill.label_features(logs,[FeatureDefinition(input_rule, "input_target")])
+    assert result
```

Comment:
Few things here:

First, best practice is to encapsulate any variables, data, etc. that are specific to your test inside the test function itself. 

In this case, you only use the logs and the input_rule inside the test_label_features_1 function, so you should move all of that inside the function (yes, you can define functions inside other functions, you just can't then use the child function outside the scope of the parent function.).

Second, is `logs` actually a list of one or more logs or is it a single log? I see you indexed on data[1] which leads me to believe it's just a single log but it should be the former.

Third, see comment below about naming conventions.

Finally, we want to validate that our label_features function not only ran successfully, but that the logic does what we expect. In this case, assuming the input_rule returns true, we expect the function to add the label we have set. Result should be a list of a logs. We then want to assert that all of the logs which match the rule have indeed been properly labeled, if we know _a priori_ which logs meet our criteria we can directly extract them using their index and then assert that the `"labels"` field is now present in them and that it contains the "input_target" label.